lib/messagebird.js

```diff
@@ -475,6 +475,29 @@ module.exports = function (accessKey, timeout, features) {
         httpRequest({ method: 'POST', path: '/verify', params: params }, callback);
       },
 
+      /**
+       * Send a verification code via email
+       *
+       * @param {String} from
+       * @param {String} to
+       * @param {Object} params
+       * @param {Function} callback
+       * @return {void}
+       */
+       createWithEmail: function (from, to, params, callback) {
```

Comment:
Can you please also add new methods to TypeScript typings in index.d.ts?

Reply:
fixed!